recipe/meta.yaml

```diff
@@ -52,12 +58,11 @@ outputs:
         - pillow >=6.2.0
         - pyparsing >=2.2.1
         - python-dateutil >=2.7
-
+        - freetype
```

Comment:
Missing empty line before `test` (for readability)

Reply:
I would say adding an empty line would make it less readable. Since this is a multi-output recipe, I would only add blank lines between each output.